core/templates/dev/head/library/Library.js

```diff
@@ -23,6 +23,10 @@ oppia.controller('Library', [
       $scope, $http, $rootScope, $window, $timeout, i18nIdService,
       urlService, CATEGORY_LIST, searchService, windowDimensionsService) {
     $rootScope.loadingMessage = 'I18N_LIBRARY_LOADING';
+    var possibleBanners = ['banner1.svg', 'banner2.svg', 'banner3.svg',
```

Comment:
Lint: break after '[' and indent by 2, viz.:

```
  var possibleBannerFilenames = [
    'banner1.svg', 'banner2.svg', 'banner3.svg', 'banner4.svg'];
  $scope.bannerImageFilename = possibleBannerFilenames[
    Math.floor(Math.random() * possibleBannerFilenames.length)];
```

(Also renamed possibleBanners to possibleBannerFilenames, since that's a more accurate description of the variable's contents.)